### DISCUSSION TITLE
Small error

### DISCUSSION THREAD

**Discussion by bigpilot** [browserengineering/book#291] [General]
I noticed one small error: it states that AJAX was invented during the "second browser wars" while the basis for AJAX (XMLHttpRequest) was actually first used in 1999.

**Comment by pavpanchekha** [comment]
Thanks, we can try to make that a bit more precise. XMLHttpRequest was originally used in Outlook Web Access as an ActiveX control in Internet Explorer (none of those technologies are around any more!) in version 5.0. The broader pattern of using XMLHttpRequest as the basis of complex web applications, however, didn't appear until the early 2000s, with GMail (2004) and GMaps (2005) being particularly big applications that depended on it. It's this widespread use that happened at the time of second browser wars.

I'll tighten up the text there when I next make a pass.

**Reply by pavpanchekha** [reply]
This is done on the repo. I'll push to the website probably when we release chapter 11, which should be coming.
